### PR TITLE
Firdos branch

### PR DESCRIPTION
**Pull request by firdos94**

### CODE REVIEW

File: .settings/org.eclipse.jdt.core.prefs [hunk 1]
```diff
@@ -1,4 +1,13 @@
 eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.annotation.inheritNullAnnotations=disabled
+org.eclipse.jdt.core.compiler.annotation.missingNonNullByDefaultAnnotation=ignore
```
Comment (NajibAbuSamra) [inline .settings/org.eclipse.jdt.core.prefs:3]: @firdos94 dear me what are all these 😮

File: src/mm/da/DataAccess.java [hunk 2]
```diff
@@ -1377,4 +1377,22 @@ public boolean editMeetingNote(int meetingId, String message)
 		return true;
 	}
 
+	@Override
+	public String getAcadimicInsById(int id) {
```
Comment (NajibAbuSamra) [inline src/mm/da/DataAccess.java:1381]: so all you want is the string? not the full object?

File: src/mm/da/DataInterface.java [hunk 3]
```diff
@@ -108,5 +108,10 @@
 
 
 	public boolean editMeetingNote(int meetingId, String message) throws SQLException;
+	
+	public String getAcadimicInsById(int id);
```
Comment (NajibAbuSamra) [inline src/mm/da/DataInterface.java:112]: same as above

File: .settings/org.eclipse.jdt.core.prefs (hunk 1 above)
Comment (firdos94) [reply .settings/org.eclipse.jdt.core.prefs:3]: i have no idea :/

File: src/mm/da/DataAccess.java (hunk 2 above)
Comment (firdos94) [reply src/mm/da/DataAccess.java:1381]: now we need just the name to add to the constructor, if you want to send full object It is also possible.

Comment (NajibAbuSamra) [reply src/mm/da/DataAccess.java:1381]: I would much prefer to send a full object whenever asked to access the db, in the future we will have to send all information and we can save work by sending it all. If you want to continue work you can complete this merge, I will make the required changes to support this choice for now.

**Review by NajibAbuSamra — APPROVED** [review]

**Pull request merged**